runtime/vm/zcinterp.m4

```diff
@@ -205,12 +206,10 @@ dnl Load the updated object pointer into R9
 dnl Extract the target address register
         LG r8,J9TR_VMThread_gsParameters_instructionAddr(J9VMTHREAD)
         LLGH r8,0(r8)
-        NILF r8,240
-
 dnl We combine a shift right by 4 and a multiply by 8 to a shift
 dnl right by 1 to get the byte offset into the save area where to
 dnl store the updated object pointer
-        SRLG r8,r8,1
+		ENCODE_RISBGN(r8,r8,57,60,63)
```

Comment:
You probably should set to set the Z bit to 1 in the I4 operand, to ensure the remaining bits of the first operand are set to zeros.  Otherwise, you might accidentally get some odd bits set for the extra bits loaded by LLGH.

Reply:
Argh yes good catch, my builds were falling all over the place and this was the cause. Thanks!

Fixed in 892129c